scripts/package.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const commander = require('commander');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const replace = require('replace-in-file');
+
```

Comment:
Add these lines so that when we add this to CI it will give a proper error code when there's an error:

```
process.on('uncaughtException', (err) => {
  console.error('Caught exception:', err)
  process.exit(1)
})

process.on('unhandledRejection', (err) => {
  console.error('Unhnadled rejection:', err)
  process.exit(1)
})
```

Reply:
After fixing please do a manual test that the error conditions are returned correctly, I think `echo %ERRORLEVEL%` on Windows (equivalent to `echo $?` on other platforms. `0` for success.